gordo/cli/workflow_generator.py

```diff
@@ -181,7 +178,23 @@ def workflow_generator_cli(gordo_ctx, **ctx):
 
     # Should we start up influx/grafana at all, i.e. is there at least one request
     # for it?"
-    context["enable_influx"] = nr_of_models_with_clients > 0
+    enable_influx = nr_of_models_with_clients > 0
+    context["enable_influx"] = enable_influx
+
+    context["postgres_host"] = f"gordo-postgres-{config.project_name}"
+
+    # If enabling influx, we setup a postgres reporter to send metadata
+    # to allowing querying about the machine from grafana
+    if enable_influx:
```

Comment:
Maybe we want to sometimes enable postgres and not influx? Could be `if enable_influx or enable_postgres:`

Reply:
That seems outside the scope of this PR.